runtime/browser/runtime.cc

```diff
@@ -301,4 +301,13 @@ void Runtime::RequestMediaAccessPermission(
       web_contents, request, callback);
 }
 
+bool Runtime::CheckMediaAccessPermission(
```

Comment:
It's not empty as commented, it was added in order to avoid flood of "Not supported" msg.